{{ cookiecutter.repo_name }}/MANIFEST.in

```diff
@@ -1,14 +1,11 @@
-include AUTHORS.rst
-include LICENSE
-include README.rst
-include requirements.txt
+graft src
+graft tests
+graft requirements
 
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
+include AUTHORS.txt LICENSE*.txt README.rst
```

Comment:
I think we don't have these, right?  Aren't these rst files now?  Please check.  If not, let's discuss.

Reply:
Thank you. We don't use .txt anymore so

`AUTHORS.rst` `LICENSE*.rst` `README.rst`

used instead.